src/Mexitek/PHPColors/Color.php

```diff
@@ -8,63 +8,63 @@
 
 namespace Mexitek\PHPColors;
 
-use \Exception;
+use Exception;
 
 /**
  * A color utility that helps manipulate HEX colors
  */
-class Color {
-
+class Color
+{
+    /**
+     * @var string
+     */
     private $_hex;
+
+    /**
+     * @var array
+     */
     private $_hsl;
+
+    /**
+     * @var array
+     */
     private $_rgb;
 
     /**
      * Auto darkens/lightens by 10% for sexily-subtle gradients.
      * Set this to FALSE to adjust automatic shade to be between given color
      * and black (for darken) or white (for lighten)
      */
-    const DEFAULT_ADJUST = 10;
+    public const DEFAULT_ADJUST = 10;
 
     /**
      * Instantiates the class with a HEX value
      * @param string $hex
-     * @throws Exception "Bad color format"
+     * @throws Exception
      */
-    function __construct( $hex ) {
-        // Strip # sign is present
-        $color = str_replace("#", "", $hex);
```

Comment:
@peter279k @mikaykun removal of this line is a breaking change - can we add it back? :)

Would be good if https://github.com/mexitek/phpColors/issues/41#issuecomment-663393993 could be done too, then my CI won't instantly fail when `dev-master` is updated heh.